### PR TITLE
Housekeeping/fix ux upgrade dep

### PR DESCRIPTION
**Pull request by mhaegeleMHP**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://github.com/porscheofficial/oss-docs/blob/67c5b24838a293ce7a964884e6005eb71f2b8579/CONTRIBUTOR_LICENSE_AGREEMENT.md)

#### References

- Preview: https://designsystem.porsche.com/issue/.../

#### Scope

Short description of the pull request scope

### CODE REVIEW

File: packages/components-angular/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
     "angular-imask": "6.4.2",
     "imask": "^6.4.3",
     "rxjs": "^7.8.0",
-    "tslib": "^2.4.1",
+    "tslib": "^2.5.0",
```
Comment (affilnost) [inline packages/components-angular/package.json:44]: I noticed that we have `"tslib": "^2.0.1"` in `/packages/components-angular/projects/angular-wrapper/package.json`. Should we update it there too?

File: packages/components-js/package.json [hunk 2]
```diff
@@ -75,18 +75,18 @@
...
+    "devtools-protocol": "^0.0.1112051",
     "globby": "^11.0.4",
     "js-beautify": "^1.14.7",
     "jsdom": "^20.0.3",
-    "local-web-server": "^5.2.1",
+    "local-web-server": "^5.3.0",
     "terser-webpack-plugin": "^5.3.6",
     "ts-node": "^10.9.1",
-    "typescript": "~4.9.4",
+    "typescript": "~4.9.5",
```
Comment (affilnost) [inline packages/components-js/package.json:87]: I noticed that we have `"typescript": "~4.6.4"` in `/packages/components-angular/package.json` - should we also update it there?

File: packages/components-vue/package.json [hunk 3]
```diff
@@ -33,32 +33,32 @@
   },
   "dependencies": {
     "@porsche-design-system/components-vue": "3.0.0-alpha.2",
-    "vue": "^3.2.45",
+    "vue": "^3.2.47",
```
Comment (affilnost) [inline packages/components-vue/package.json:36]: I noticed that we have: `"vue": "^2.7.10",` in `/packages/storefront/package.json` and `"vue": ">=3.0.0 <4.0.0"` in `/packages/components-vue/projects/vue-wrapper/package.json` - should we also update it there?

File: packages/components/package.json [hunk 4]
```diff
@@ -47,31 +47,31 @@
     "@porsche-design-system/utilities-v2": "0.0.0",
     "@splidejs/splide": "^4.1.4",
     "@stencil/core": "~2.20.0",
-    "jss": "^10.9.2",
+    "jss": "^10.10.0",
```
Comment (affilnost) [inline packages/components/package.json:50]: I noticed that we have `"jss": "^10.9.2",` in `/packages/utilities/projects/styles/package.json` - should we also update it there?

File: packages/components/package.json [hunk 5]
```diff
@@ -47,31 +47,31 @@
...
     "jss-plugin-sort-css-media-queries": "^1.0.1-beta.0",
-    "jss-preset-default": "^10.9.2",
+    "jss-preset-default": "^10.10.0",
```
Comment (affilnost) [inline packages/components/package.json:52]: I noticed that we have `"jss-preset-default": "^10.9.2",`, in `/packages/utilities/projects/styles/package.json` - should we also update it there?

File: packages/components-vue/package.json (hunk 3 above)
Comment (mhaegeleMHP) [reply packages/components-vue/package.json:36]: There is a difference between the vue package which contains the vue 3 wrappers and our storefront, which is still on vue 2. See dependencies .md file.

File: packages/components-js/package.json (hunk 2 above)
Comment (mhaegeleMHP) [reply packages/components-js/package.json:87]: Angular decides which typescript versions are allowed, depending on the version of angular we use. Thats why its pinned. See dependencies.md.

File: packages/components-angular/package.json (hunk 1 above)
Comment (mhaegeleMHP) [reply packages/components-angular/package.json:44]: They are upgraded as we can see in the yarn.lock. It seems they got previously upgraded via deletion of yarn.lock thats why its not mirrored in the package.json.

File: packages/components/package.json (hunk 4 above)
Comment (mhaegeleMHP) [reply packages/components/package.json:50]: See comment in angular

File: packages/components/package.json (hunk 5 above)
Comment (mhaegeleMHP) [reply packages/components/package.json:52]: See comment in angular

**Review by denyo — APPROVED** [review]

**Pull request merged**